scoring.js

```diff
@@ -0,0 +1,57 @@
+const calculateScore = (player) => {
+  switch (player.position) {
+    case 'QB':
+      return (
+        player.stats.passing.yards / 25 +
+        player.stats.passing.touchdowns * 6 +
+        player.stats.passing.interceptions * -3 +
+        player.stats.rushing.yards / 10 +
+        player.stats.rushing.touchdowns * 6 +
+        player.stats.rushing.fumbles * -3
+      )
+    case 'RB':
+      return (
+        player.stats.rushing.yards / 10 +
+        player.stats.rushing.touchdowns * 6 +
```

Comment:
seems touchdown points are 6 always... maybe we should move the same points to a variable.